src/formats/SDF.cpp

```diff
@@ -40,6 +40,10 @@ SDFFormat::SDFFormat(std::string path, File::Mode mode, File::Compression compre
             steps_positions_.push_back(position);
         }
     }
+
+    if (mode == File::READ && steps_positions_.empty()) {
```

Comment:
Why is this needed ? `Trajectory::read` should throw an error instead if the file contains 0 steps

Reply:
Ok, I've changed this.